webpack/common.config.js

```diff
@@ -37,12 +37,15 @@ const base = {
         ]
       },
       {
-        test: /(fonts|files)\/.*\.(svg|woff2?|ttf|eot|otf)(\?.*)?$/,
+        test: /\.(svg|woff2?|ttf|eot|otf)$/,
+        type: 'javascript/auto',
+        include: /fonts/,  // This line ensures only files in the 'fonts' folder are processed
         loader: 'file-loader',
         options: {
           name: '[name].[ext]',
-          outputPath: 'fonts',
-          postTransformPublicPath: (p) => `'../' + ${p}`
+          outputPath: 'fonts/',  // Output directory for the fonts within the output path of webpack
+          publicPath: '../fonts/',  // Sets correct public path to reference fonts in the CSS
```

Comment:
Are you sure this change works? It took me quite some time to figure the `postTransformPublicPath` thing out. The paths need to work when rdmo is run under `https://example.com/rdmo`.

Reply:
eh, good question, I don't have a way to test this base_url setup so it might be broken like this if that `postTransformPublicPath` was specifically for that case..

Reply:
I can test that at home, but I will be back on Friday.

Reply:
yes, thanks that would be great.
It could be helpful If there is a way to run that setup locally or in end-to-end tests and take screenshots I think..

Reply:
maybe this snippet helps ? generated code...
```
          publicPath: (url, resourcePath, context) => {
            // Adjust the public path based on whether BASE_URL is set
            const baseUrl = process.env.BASE_URL || ''
            return `/${baseUrl}/static/fonts/`
          },
          postTransformPublicPath: (p) => `__webpack_public_path__ + ${p}`,
```

Reply:
nope, we don't have `process.env.BASE_URL` at compile time because we compile it for *all* the RDMO instances 😄

Reply:
Why was the inital code changed in the first place?

Reply:
something got broken with the icons after update of `css-loader` ? https://github.com/rdmorganiser/rdmo/pull/886#issuecomment-2049160820

Reply:
maybe my changes should just be reverted and only the  `type: "javascript/auto"` and `esModule: false` should be set as changes to this part

Reply:
Ive committed this ⬆️ snippet , think that resolves this conversation

Reply:
Hey cool I had the problem in another project and the combination of `type` and `esModule` fixed it. The change is probably to create different bundles per build.